tests/roles/run_tests/vars/main.yml

```diff
@@ -5,7 +5,7 @@ NAMESPACE: "{{ lookup('env', 'NAMESPACE') | default('metal3', true) }}"
 IRONIC_NAMESPACE: "{{ lookup('env', 'IRONIC_NAMESPACE') | default('baremetal-operator-system', true) }}"
 NAMEPREFIX: "{{ lookup('env', 'NAMEPREFIX') | default('baremetal-operator', true) }}"
 CRS_PATH: "{{ metal3_dir }}/tests/roles/run_tests/templates"
-TEMP_GEN_DIR: "{{ metal3_dir }}/vm-setup/roles/run_tests/files/manifests"
+TEMP_GEN_DIR: "{{ metal3_dir }}/tests/roles/run_tests/files/manifests"
 MANIFESTS_DIR: "{{ metal3_dir }}/vm-setup/roles/run_tests/files"
```

Comment:
what are these folders for and why one `MANIFESTS_DIR` is different from the other `TEMP_GEN_DIR` with this change?

Reply:
MANIFESTS_DIR was updated already and it is the path to all manifests I think and TEMP_GEN_DIR is one contains the generated templates

Reply:
but the paths are same, we do not have `run_tests` folder under `vm-setup/roles` anymore AFAIU, why the `MANIFESTS_DIR` should not be changed?

Reply:
nice catch, I will check this. I was thinking that  already under `tests`

Reply:
That var `MANIFESTS_DIR` is not used anywhere so I think we do not needed ! removed is that ok ?